rules/cypress.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: ['cypress'],
+  extends: ['plugin:cypress/recommended'],
+  rules: {
+    'cypress/no-unnecessary-waiting': 'warn'
```

Comment:
This seems to be new, right?

Reply:
yes, this a rule is to when use wait in cypress https://github.com/cypress-io/eslint-plugin-cypress#disable-rules

Reply:
Okidoki. It is fine to define rules as long as it does require changes on the cypress tests when we introduce this package on the memberhub